app/models/local_authority.rb

```diff
@@ -34,6 +34,10 @@ class LocalAuthority < ApplicationRecord
 
   before_update :set_active
 
+  def self.notify_notion_link
```

Comment:
Why does this need to be a method?
Can't it just live directly in the yml file if it's not going to change.

Reply:
I don't think we're consistent about this. For example, we have links to GOV.UK advice, legislation, planning application fees, and Notify support in the views. We have other legislation, council documents, and Notify documentation in the locales.

Reply:
I'd still argue we should be putting static links or content that doesn't change based on application logic in the locale unless it's dynamic like `map_link` but yeah there doesn't seem to be all that much consistency.

Reply:
> I'd still argue we should be putting static links or content that doesn't change based on application logic in the locale

Yep — I didn't mean to sound like I was disagreeing with you, more that we should aim to increase consistency where possible.

Best to avoid having any content in the views really, we should try to put it all in the locales, and I think that should include URLs as well as the displayed text.